packer/install-dependencies.sh

```diff
@@ -361,19 +361,27 @@ K8S_VERSIONS="
 1.12.7
 1.12.6
 1.11.9
+1.11.9-azs
 1.11.8
+1.11.8-azs
 1.10.13
 1.10.12
 1.9.11
 1.9.10
 "
 for KUBERNETES_VERSION in ${K8S_VERSIONS}; do
-    HYPERKUBE_URL="k8s.gcr.io/hyperkube-amd64:v${KUBERNETES_VERSION}"
-    extractHyperkube "docker"
-    CONTAINER_IMAGE="k8s.gcr.io/cloud-controller-manager-amd64:v${KUBERNETES_VERSION}"
-    pullContainerImage "docker" ${CONTAINER_IMAGE}
-    echo "  - ${HYPERKUBE_URL}" >> ${RELEASE_NOTES_FILEPATH}
-    echo "  - ${CONTAINER_IMAGE}" >> ${RELEASE_NOTES_FILEPATH}
+    if [[ $KUBERNETES_VERSION == *"azs"* ]]; then
+      HYPERKUBE_URL="msazurestackdocker/hyperkube-amd64:v${KUBERNETES_VERSION}"
+      extractHyperkube "docker"
+      echo "  - ${HYPERKUBE_URL}" >> ${RELEASE_NOTES_FILEPATH}
```

Comment:
these two lines 
```
extractHyperkube "docker"
echo "  - ${HYPERKUBE_URL}" >> ${RELEASE_NOTES_FILEPATH}
```
Can be extracted from the if/else statement since they are repeated in both.